src/main/java/com/product/controller/ProductController.java

```diff
@@ -31,25 +31,25 @@ public ProductController(ProductService productService) {
 	}
 
 	@GetMapping("/products")
-	public List<Product> getProduct() {
-		return productService.getAllProducts();
+	public List<Product> getProduct(@RequestParam(name = "order") String order,@RequestParam(name = "containName") String containName) {
```

Comment:
You can also DIRECTLY take in `Order order` instead of `String order`.

That way the if condition in ur Service, would look way cleaner 

Right now it is ..
```
if (order.equalsIgnoreCase((Order.ASC).toString()))
```

otherwise it can be 
```
 order == Order.ASC
```

Reply:
Done.